Micro/src/main/java/com/revature/Micro/util/JwtUtil.java

```diff
@@ -54,6 +54,24 @@ public String generateToken(UserDetails userDetails) {
         return createToken(claims, userDetails.getUsername());
     }
 
+    public String generateTempToken(UserDetails userDetails)
+    {
+        Map<String, Object> claims = new HashMap<>();
+        return createTempToken(claims, userDetails.getUsername());
+    }
+
+    private String createTempToken(Map<String, Object> claims, String subject)
+    {
+        Date now = new Date(System.currentTimeMillis());
+        Date expire = new Date(System.currentTimeMillis() + 1000*60*2); // one hour from issue time
```

Comment:
change comment to accurately depict expiration time (2 minutes)

Reply:
fixed